### PR TITLE
Add VanillaDeathChest documentation

### PR DESCRIPTION
**Pull request by TheRandomLabs**

### CODE REVIEW

File: docs/Mods/VanillaDeathChest/Death_Chest_Defense.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## Importing the package
+`import mods.vanilladeathchest.DeathChestDefense;`
+
+## Unlocker item
+```
+//DeathChestDefense.setUnlocker(string stage, IItemStack unlocker);
+DeathChestDefense.setUnlocker("example_stage", <minecraft:diamond_axe>);
+```
+A consumption/damage amount can also be set.
```
Comment (kindlich) [inline docs/Mods/VanillaDeathChest/Death_Chest_Defense.md]: Could you elaborate on this one?
Or just add an example?

File: docs/Mods/VanillaDeathChest/Death_Chest_Defense.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+DeathChestDefense.setDamageUnlockerInsteadOfConsume(string stage, bool flag);
+DeathChestDefense.setDamageUnlockerInsteadOfConsume("example_stage", true);
+```
+
+## Unlock failed chat message
+```
+//DeathChestDefense.setUnlockFailedChatMessage(string stage, string message);
+DeathChestDefense.setUnlockFailedChatMessage("example_stage", "You need to get a %2$ to unlock your chest!");
+```
+The string takes two arguments: the amount and display name of the required items.
```
Comment (kindlich) [inline docs/Mods/VanillaDeathChest/Death_Chest_Defense.md]: Could you specify the placeholder `%2$` more thoroughly?

**Comment by TheRandomLabs** [comment]
The strings are not localizable because they are server-sided.

**Comment by TheRandomLabs** [comment]
I think I've fixed everything.

**Comment by TheRandomLabs** [comment]
@kindlich Can you review the changes?

**Pull request merged**